metrics/ewma.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sync/atomic"
 )
 
-// EWMAs continuously calculate an exponentially-weighted moving average
+// EWMA methods continuously calculate an exponentially-weighted moving average
```

Comment:
It's an interface, not methods. `EWMA continuously calculates [...]` is enough

Reply:
Thank you - updated.